docs/world maintainer.md

```diff
@@ -35,7 +35,7 @@ When a world is unmaintained, the [core maintainers](https://github.com/orgs/Arc
 can vote for a new maintainer if there is a candidate.
 For a vote to pass, the majority of participating core maintainers must vote in the affirmative.
 The time limit is 1 week, but can end early if the majority is reached earlier.
-Voting shall be conducted on Discord in #archipelago-dev.
+Voting shall be conducted on Discord in #ap-world-dev.
```

Comment:
I'm not sure if this should be core instead 🤔 opinions?

Reply:
Not a strong one

I do see your point about it being a core thing so it should be in ⁠ap-core-dev 

But then other other hand, it's something where the discussion would probably start in ⁠ap-world-dev, so we'd have to "move over" at some point for the poll I guess?

Reply:
No idea, I'd be fine with either, however I think previous communications of that nature were done in a thread that was not visible to the public?

Reply:
That's also true haha